packages/sdk-auth/src/auth.js

```diff
@@ -1,6 +1,5 @@
 /* @flow */
-/* global fetch Request Headers */
-
+// global fetch Request Headers
```

Comment:
Is this some new rule to eslint regarding comments or why are we making this change?

Reply:
When `yarn flow` is run, `request` and `headers` are both reported as an error  and sometimes it does happen too on CI as well.

Reply:
I would suggest digging into this one a bit deeper. What did the error say and why are we having them there inside a comment? It looks like a configuration setting to me.

Reply:
It is eslint change of rules usage [see here](https://eslint.org/docs/rules/no-unused-vars).

Reply:
Sorry @Babanila but I am not following here. What is the change of rule?

Reply:
The global variables `Request` and `Headers` was defined but were not used in the file,  setting them in the eslint configuration file to avoid the error report [docs link](https://eslint.org/docs/user-guide/configuring#specifying-globals).